components/dot_menu/dot_menu.tsx

```diff
@@ -49,7 +50,7 @@ type Props = {
     enableEmojiPicker?: boolean; // TechDebt: Made non-mandatory while converting to typescript
     channelIsArchived?: boolean; // TechDebt: Made non-mandatory while converting to typescript
     currentTeamUrl?: string; // TechDebt: Made non-mandatory while converting to typescript
-    appBindings?: AppBinding[];
+    appBindings?: AppBinding[] | null;
```

Comment:
do we need to pass the null check here

Reply:
Actually, appBindings will always be defined. Here null will have a special meaning, since it will not be the same receiving an empty list (no bindings) or a null (you must fetch the bindings).